### ISSUE TITLE
Show timestamp in skype 7 and NVDA report

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4951]
**Reported by chrislm on 2015-02-26 14:45**
When the check  box to report time in Skype options is disable, NVDA announce  evenly date and time of instant messages.
Note: this does not happen with the interface of Skype in English, I noticed this with spanish and italian interface of Skype.
Probably this also happens with other languages...

**Comment by leonardder** [comment]
I can reproduce with the Dutch version of skype 7.38, also when i set the language back to English. However, this issue is also visible in Narrator. Probably a bug in Skype in how messages are exposed. Would it be possible to file a bug against Microsoft for Skype for desktop?

**Comment by Adriani90** [comment]
@chrislm, @florianionascu7, @ruifontes can anybody reproduce this in Skype 8 or Skype UWP?

**Comment by k-kolev1985** [comment]
There is no option in Skype 8 or Skype UWP to remove timestamps from chat messages, so this issue is irrelevant for them.

**Comment by Adriani90** [comment]
@josephsl how do you see the support for skype 7? If it is completely dropped I suggest to close it after 4 weeks or so.

**Comment by josephsl** [comment]
Hi,

I think we should keep it open until NVDA 2019.1 is released, during which time folks using Skype 7 should be invited to give comment. After that, I suggest closing it, as Skype 7 is no longer supported by Microsoft.

Thanks.

**Comment by Adriani90** [comment]
No one using Skype 7 commented on this, so I guess most people already changed to Skype 8. I am closing this as won't fix. Please coment on it if you don't agree and we can reopen the discussion.
